src/content/blog/idx-official-online-editor-partner.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Google IDX: Our Official Online Editor Partner"
+authors:
+  - thuy
+description: |
+  We are happy to announce that Google IDX has become Astro’s official online editor partner, donating $10,000 each month towards the ongoing open source maintenance and development of Astro.
```

Comment:
Here its being used as an adjective describing Astro's maintenance and development workflow, so it should include a hyphen
```suggestion
  We are happy to announce that Google IDX has become Astro’s official online editor partner, donating $10,000 each month towards the ongoing open-source maintenance and development of Astro.
```